src/components/LHNOptionsList/OptionRowLHN.js

```diff
@@ -0,0 +1,226 @@
+import _ from 'underscore';
+import lodashGet from 'lodash/get';
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+    TouchableOpacity,
+    View,
+    StyleSheet,
+} from 'react-native';
+import styles from '../../styles/styles';
+import * as StyleUtils from '../../styles/StyleUtils';
+import Icon from '../Icon';
+import * as Expensicons from '../Icon/Expensicons';
+import MultipleAvatars from '../MultipleAvatars';
+import Hoverable from '../Hoverable';
+import DisplayNames from '../DisplayNames';
+import IOUBadge from '../IOUBadge';
+import colors from '../../styles/colors';
+import withLocalize, {withLocalizePropTypes} from '../withLocalize';
+import Text from '../Text';
+import SubscriptAvatar from '../SubscriptAvatar';
+import CONST from '../../CONST';
+import * as ReportUtils from '../../libs/ReportUtils';
+import variables from '../../styles/variables';
+import themeColors from '../../styles/themes/default';
+import SidebarUtils from '../../libs/SidebarUtils';
+
+const propTypes = {
+    /** Style for hovered state */
+    // eslint-disable-next-line react/forbid-prop-types
+    hoverStyle: PropTypes.object,
+
+    /** The ID of the report that the option is for */
+    reportID: PropTypes.string.isRequired,
+
+    /** Whether this option is currently in focus so we can modify its style */
+    isFocused: PropTypes.bool,
+
+    /** A function that is called when an option is selected. Selected option is passed as a param */
+    onSelectRow: PropTypes.func,
+
+    /** Toggle between compact and default view */
+    viewMode: PropTypes.oneOf(_.values(CONST.OPTION_MODE)),
+
+    style: PropTypes.oneOfType([PropTypes.arrayOf(PropTypes.object), PropTypes.object]),
+
+    ...withLocalizePropTypes,
+};
+
+const defaultProps = {
+    hoverStyle: styles.sidebarLinkHover,
+    viewMode: 'default',
+    onSelectRow: () => {},
+    isFocused: false,
+    style: null,
+};
+
+const OptionRowLHN = (props) => {
+    const optionItem = SidebarUtils.getOptionData(props.reportID);
+    if (!optionItem) {
+        return null;
+    }
+
+    let touchableRef = null;
+    const textStyle = props.isFocused
+        ? styles.sidebarLinkActiveText
+        : styles.sidebarLinkText;
+    const textUnreadStyle = optionItem.isUnread
+        ? [textStyle, styles.sidebarLinkTextUnread] : [textStyle];
+    const displayNameStyle = StyleUtils.combineStyles(props.viewMode === CONST.OPTION_MODE.COMPACT
+        ? [styles.optionDisplayName, ...textUnreadStyle, styles.optionDisplayNameCompact, styles.mr2]
+        : [styles.optionDisplayName, ...textUnreadStyle], props.style);
+    const alternateTextStyle = StyleUtils.combineStyles(props.viewMode === CONST.OPTION_MODE.COMPACT
+        ? [textStyle, styles.optionAlternateText, styles.textLabelSupporting, styles.optionAlternateTextCompact]
```

Comment:
Turns out that on compact (focus) mode, because of the `styles.optionAlternateText` we had this emoji alignment issue https://github.com/Expensify/App/issues/41367 on iOS: Native only.